test/components/general/progressbar.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { shallow } from "enzyme";
+import ProgressBar from "../../../src/components/general/ProgressBar";
+
+describe("ProgressBar component", () => {
+  it("renders the percentage as text", () => {
+    let percentage = 30;
+    const wrapper = shallow(<ProgressBar progressPercentage={percentage}></ProgressBar>);
+    expect(wrapper.text()).toContain(`${percentage}%`);
```

Comment:
Nitpick: IMO, we should hard-code `30%` here. For example, imagine we changed the interface to accept a fraction between 0 and 1—we wouldn't want to render `.3%`.